storage/storage-interface/src/lib.rs

```diff
@@ -390,11 +390,11 @@ pub trait DbReader: Send + Sync {
 }
 
 impl MoveStorage for &dyn DbReader {
-    fn batch_fetch_resources(&self, access_paths: Vec<AccessPath>) -> Result<Vec<Vec<u8>>> {
-        self.batch_fetch_resources_by_version(access_paths, self.fetch_synced_version()?)
+    fn fetch_resources(&self, access_paths: Vec<AccessPath>) -> Result<Vec<Vec<u8>>> {
+        self.fetch_resource_at_version(access_paths, self.fetch_synced_version()?)
```

Comment:
This also needs to be updated to take in only a single access_path, as opposed to multiple access paths, i.e.,

```
fn fetch_resource(&self, access_path: AccessPath) -> Result<Vec<u8>> {
```

Reply:
Also, change `fetch_resources` -> `fetch_resource`

Reply:
Done ! 👍